packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/index.js

```diff
@@ -7,45 +7,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
-import { get, isEmpty, map, size, sortBy } from 'lodash';
+import { get, snakeCase, isEmpty, map, sortBy } from 'lodash';
 
 import LeftMenuLink from 'components/LeftMenuLink';
 
 import styles from './styles.scss';
 import messages from './messages.json';
 
 function LeftMenuLinkContainer({ plugins }) {
+  const pluginsObject = plugins.toJS();
+
   // Generate the list of sections
-  const linkSections = map(plugins.toJS(), plugin => (
-    plugin.leftMenuSections.map((leftMenuSection, j) => {
+  const pluginsSections = Object.keys(pluginsObject).reduce((acc, current) => {
+    pluginsObject[current].leftMenuSections.forEach((section = {}) => {
+      if (!isEmpty(section.links)) {
+        acc[snakeCase(section.name)] = {
+          name: section.name,
+          links: (get(acc[snakeCase(section.name)], 'links') || []).concat(section.links.map(link => {
```

Comment:
Use default get value

```js
get(acc[snakeCase(section.name)], 'links', []).concat(....
```

Reply:
No we can't because you can't do: `undefined.concat(....)`